javascript/doc-structure/article-2022.js

```diff
@@ -227,7 +227,11 @@ var modCredit = ''
 if (f.modifiers && f.modifiers != '') modCredit = s.modifiedBy+' '+f.modifiers+s.sentenceDelimiter
 
 var translatorCredit = ''
-if (g.isTranslation) translatorCredit = s.translatedBy+' '+f.translators+s.sentenceDelimiter
+if (g.isTranslation) {
+	// Remove the space in Chinese
```

Comment:
An alternative approach would be to include the space in s.translatedBy.  It wouldn't take a lot of work to change the code, but it would avoid the need to extend the list of exceptions for other languages, give more flexibility, and reduce the code complexity.

Reply:
It would also solve the problem for all these other cases:

````
s.author = "By:" // followed by name of author(s)
s.previousAuthors = "Previously by:" // followed by name of previous author(s), before substantive changes were made by the current author
s.modifiedBy = "Changed by:" // person's name appears after colon
````

(there may be some more)

I'd suggest changing the boilerplate text files to say

````
s.translatedBy = "Translator:"+" "
````
For Chinese that would be +"".

I can make this change, if you like. You'd need to remove the changes from this PR.

Reply:
Fixed. Thanks.